src/open_producten/producttypen/tests/api/test_producttype_filters.py

```diff
@@ -497,3 +502,39 @@ def test_verbruiksobject_schema_filter(self):
             response.data["results"][0]["verbruiksobject_schema"]["naam"],
             "parkeer-verbruik-schema",
         )
+
+    def test_zaaktype_uuid_filter(self):
+        uuid = uuid4()
+
+        ZaakTypeFactory.create(uuid=uuid)
+        ZaakTypeFactory.create()
+
+        response = self.client.get(self.path, {"zaaktypen__uuid": uuid})
```

Comment:
If this is not a requirement for these filters, you can ignore it, but I was wondering: does this filter only allow single UUIDs, or would it be possible to do something like `zaaktypen__uuid__in` to filter on multiple as well?